apps/src/gamelab/animationListModule.js

```diff
@@ -194,6 +194,8 @@ export function setInitialAnimationList(serializedAnimationList) {
       } else {
         animation.frameDelay = 2;
       }
+    } if (animation.looping === undefined) {
+      animation.looping = true;
```

Comment:
I think this `if` statement should go on its own line, so it doesn't get confused with an `else if`.